LeadAlerts.Web/Domain/MessageSender.cs

```diff
@@ -1,28 +1,28 @@
 ﻿using Twilio;
+using Twilio.Types;
+using Twilio.Rest.Api.V2010.Account;
+using System.Threading.Tasks;
 
 namespace LeadAlerts.Web.Domain
 {
     public interface IMessageSender
     {
-        Message Send(string message);
+        Task<MessageResource> SendAsync(string messageStr); 
```

Comment:
As a good practice we don't use the type of the variable as a postfix token (that was common when you use Hungarian notation). This parameter should be `message`. Does it make sense to you?